docs: add Sass deprecations notice in docs (#41283)
twbs/bootstrap: site/content/docs/5.3/getting-started/vite.md

     },
     server: {
       port: 8080
     },

>      // optional: silencing Sass deprecation warnings, see note below

Reviewer: ```suggestion
     // Optional: Silence Sass deprecation warnings. See note below.
```

     css: {
        preprocessorOptions: {
           scss: {
             silenceDeprecations: [
               'import',
               'mixed-decls',
               'color-functions',
               'global-builtin',
             ],
           },
        },
     },
   }
   ```

>    ***Note:*** *Sass deprecation warnings are shown when compiling source Sass files with the last versions of Dart Sass. This does not prevent compilation or usage. We are [working on a long-term fix]({{< param repo >}}/issues/40962), in the meantime the above config will suppress these warnings.*
> 

Reviewer: ```suggestion
   **Note:** Sass deprecation warnings are shown when compiling source Sass files with the latest versions of Dart Sass. This does not prevent compilation or usage of Bootstrap. We're [working on a long-term fix]({{< param repo >}}/issues/40962), but in the meantime these deprecation notices can be ignored.
```
